### PR TITLE
Add help command to DG

### PR DESCRIPTION
**Pull request by yarnmengnus**

### CODE REVIEW

**Review by jack1e0 — APPROVED** [review]
lgtm

**Review by eyelessrhyme7 — APPROVED** [review]
LGTM! Good job following structure and forking workflow.

**Review by newway1814 — APPROVED** [review]
LGTM!

Nice job!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -240,6 +240,37 @@ _{more aspects and alternatives to be added}_
...
+
+* **Alternative 2: Store the Details in each command.**
+
+  * Pros: Centralised area to view all Messages.
+
+  * Cons: Does not follow OOP. Future implementations that tap on these Details will take longer to implement.
+
+
+
+
```
Comment (kiatkat) [inline docs/DeveloperGuide.md:273]: Nice.

**Review by kiatkat — APPROVED** [review]
LGTM.

**Pull request merged**